src/store/channels-list/saga.test.ts

```diff
@@ -307,12 +321,20 @@ describe(fetchConversations, () => {
     await expectSaga(fetchConversations)
       .provide([
         [
-          matchers.call.fn(fetchConversationsApi),
-          MOCK_CHANNELS,
+          matchers.call(chat.get),
+          chatClient,
```

Comment:
I think these tests could use a follow up cleaning similar to #988. Basically, we shouldn't need to clutter the tests with the default response of the mock client.

Also just a note: if you take the trailing comma off these arrays the formatter will pull them up to a single line (if they're short enough) which is far easier to read.

Reply:
Nice okay will do a follow up PR!